org.gnome.moserial.yaml

```diff
@@ -0,0 +1,30 @@
+id: org.gnome.moserial
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: moserial
+finish-args:
+  # moserial requires access to serial devices connected to the host.
+  - --device=all
+  - --filesystem=home
+  - --socket=x11
```

Comment:
if you're using x11 then you should --share=ipc as well to use accelerated shared memory.

Further, if you support wayland then we encourage using --socket=fallback-x11 which will prefer wayland if available, sharing just x11 will run in xwayland for most wayland users.